gameplay/src/Matrix.cpp

```diff
@@ -906,12 +906,12 @@ void Matrix::transformPoint(const Vector3& point, Vector3* dst) const
 void Matrix::transformVector(Vector3* vector) const
 {
     GP_ASSERT(vector);
-    transformVector(vector->x, vector->y, vector->z, 0.0f, vector);
+    transformVector(vector->x, vector->y, vector->z, 1.0f, vector);
```

Comment:
Can you explain this change? This method is supposed to be for transforming a direction vector. The method comments even say: `Transforms the specified vector by this matrix by treating the fourth (w) coordinate as zero.`